requirements-test.txt

```diff
@@ -1,9 +1,5 @@
 datadog
 freezegun==0.3.11
-msgpack-python<0.5.0
 pytest-cov>=2.5.1,<2.6.0
-pytest-timeout==1.2.1
```

Comment:
Let's keep it around. Even if we're not using it, we should be to prevent tests from having huge timeouts and taking hours to run (this happened recently)

Reply:
What tests? During py3 it might be a good opportunity to start decorating those with `pytest.mark.timeout`. I'm inclined towards removing it here though, so when we do add it back, it'll be more intentional.

Reply:
I'd be more inclined to just set a default timeout to something low, like 10-15 seconds. Then any tests that need to be longer than that can explicitly decorate themselves with higher timeout.